docs/Secure-Coding-Guide-for-Python/CWE-707/CWE-175/example03.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
```

Comment:
I believe that `example03.py` can be simplified and that we should keep CLI interaction to a minimum. Its enough to experience that a 12.345 with a German local results in 12345.0.

```suggestion
# SPDX-FileCopyrightText: OpenSSF project contributors
# SPDX-License-Identifier: MIT
""" Non-compliant Code Example """
import locale
 
def compare_number(number):
    input_number = locale.atof(input(f"Enter a number {number}: "))
    print(f"Locale is {locale.getlocale()}, you entered {input_number}.")
    print(f"Does the number {number} match {input_number}? {number == input_number}")

locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
compare_number(12.345)

```